security/sanitizers.rst

```diff
@@ -0,0 +1,442 @@
+.. _security_sanitizers:
+
+Sanitizers
+==========
+
+.. warning::
+
+   Do not use sanitizers for production builds, especially for binaries with elevated privileges (e.g., SUID).
+   Sanitizer runtimes rely on environment variables and can enable privilege escalation.
+   Use only in development and testing.
+
+Sanitizers are powerful runtime instrumentation tools that detect issues such as:
+
+* Buffer overflows (stack/heap), use-after-free, double-free
+* Data races in multithreaded code
+* Memory leaks
+* Use of uninitialized memory
+* A wide range of undefined behaviors
+
+Compilers such as GCC, Clang, and MSVC support sanitizers via compiler and linker flags.
+
+This page explains recommended approaches for integrating compiler sanitizers into your workflow with Conan.
+
+Compiler Sanitizer Support Comparison
+-------------------------------------
+
+.. important::
+
+   Always rebuild all dependencies from source when enabling sanitizers to ensure consistent instrumentation
+   and to avoid false positives (particularly critical for MemorySanitizer).
+
+Each compiler has different levels of support for various sanitizers, Clang being the most comprehensive so far.
+To help you choose the right sanitizer for your needs and compiler, here is a summary of the most common ones:
+
++----------------------------------------+-----+-------+------+-----------------------------------------+
+| Sanitizer                              | GCC | Clang | MSVC | Notes                                   |
++========================================+=====+=======+======+=========================================+
+| **AddressSanitizer (ASan)**            | ✅  | ✅    | ✅   | MSVC: Not supported for 32-bit targets  |
```

Comment:
For this one, checking the docs looks like it's supported for both x86 and x64 on Windows 10 and later: https://learn.microsoft.com/en-us/cpp/sanitizers/asan?view=msvc-170